app/src/test/java/com/courseproject/tindar/usecases/userlist/UserListInteractorUnitTest.java

```diff
@@ -0,0 +1,36 @@
+package com.courseproject.tindar.usecases.userlist;
+
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class UserListInteractorUnitTest {
+    private static final String USER_ID_1 = "1";
+    private static final String USER_ID_2 = "2";
+    private static final ArrayList<String> USERLIST = new ArrayList<>(Arrays.asList(USER_ID_1, USER_ID_2));
+
+    private static class MockUserListDsGateway implements UserListDsGateway {
+
+        @Override
+        public ArrayList<String> getAllOtherUserIds(String userId) {
+            return USERLIST;
+        }
+    }
+
+    @Test
+    public void getUserListOne() {
+        MockUserListDsGateway userListDsGateway = new MockUserListDsGateway();
+        assertEquals(USERLIST, userListDsGateway.getAllOtherUserIds(USER_ID_1));
```

Comment:
DsGateway should be used as an input argument to instantiate the interactor. and the method in the interactor should be called. Due to lack of time, I will just make a direct contribution to this PR.